docs/guides/event_loop.md

```diff
@@ -1,20 +1,47 @@
 (intro-to-event-loop)=
```

Comment:
Why isn't this file under `howtos/extending/` like the magicgui one?

Reply:
Sorry, see: https://github.com/napari/docs/pull/286#discussion_r1418490112 @DragaDoncila suggested that this remain where it is. I will update my OP to reflect

Reply:
That's fine, then just put it back in the _toc.yml

Reply:
Ah whoops, forgot to update the toc! Good pick up!